src/main.cpp

```diff
@@ -1080,7 +1134,9 @@ bool ContextualCheckTransaction(
 
     if (!tx.vJoinSplit.empty() ||
         !tx.vShieldedSpend.empty() ||
-        !tx.vShieldedOutput.empty())
+        !tx.vShieldedOutput.empty() ||
+        orchard_bundle.SpendsEnabled() ||
```

Comment:
We know by construction that if an orchard bundle is present that it contains a nonempty list of actions, so it's okay to just check the flags here, but I'd appreciate a comment to this effect; I had forgotten whether that was the case and had to go look it up.